base/get_database.py

```diff
@@ -133,3 +133,8 @@ def get_usercache_db():
     current_db = MongoClient().Stage_database
     UserCache = current_db.Stage_usercache
     return UserCache
+
+def get_fake_trips_db():

```

Comment:
For the unit tests, I am not sure whether these should be stored into the Fake_Trips_db. The rest of the code will assume that they are in the section Db like real sections. Might be a good idea to allow users to specify which DB they want the fake trips to go into.